.github/workflows/pulp_ci.yml

```diff
@@ -70,6 +97,7 @@ jobs:
           sudo kubectl logs -l app=pulp-resource-manager --tail=10000
           sudo kubectl logs -l app=pulp-worker --tail=10000
   deploy:
+    if: github.event_name != 'pull_request'
     needs: pulp
```

Comment:
this is a problem if we want a job that depends on others:
https://github.community/t5/GitHub-Actions/How-do-I-specify-job-dependency-running-in-another-workflow/m-p/33972#M1736
So in this case, if we had a workflow for `docs` and another for `pulp`, it would not be possible having deploy depending on both `docs` and `pulp`